R/webshot.R

```diff
@@ -287,7 +292,9 @@ new_session_screenshot <- function(
       }
     })$
     then(function(value) {
-      message(url, " screenshot completed")
+      if (verbose) {
+        message(url, " screenshot completed")
+      }
```

Comment:
If we update the argument name to `quiet`

```suggestion
      if (!isTRUE(quiet)) message(url, " screenshot completed")
```